ast/builtins.go

```diff
@@ -1109,6 +1110,19 @@ var Contains = &Builtin{
 	Categories: stringsCat,
 }
 
+var StringCount = &Builtin{
+	Name:        "strings.count",
+	Description: "Returns the number of non-overlapping instances of a substring in a string.",
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("cheese", types.S).Description("string to search in"),
+			types.Named("e", types.S).Description("substring to look for"),
```

Comment:
Sorry I didn't notice this before: can we please change the names, like
```suggestion
			types.Named("search", types.S).Description("string to search in"),
			types.Named("substring", types.S).Description("substring to look for"),
```

Reply:
@srenatus Thank you, Updated accordingly!